cupy/random/distributions.py

```diff
@@ -305,6 +305,33 @@ def standard_exponential(size=None, dtype=float):
     return rs.standard_exponential(size, dtype)
 
 
+def standard_gamma(shape, size=None, dtype=float):
+    """Standard gamma distribution.
+
+    Returns an array of samples drawn from the standard gamma distribution. Its
+    probability density function is defined as
+
+    .. math::
+       f(x) = \\frac{1}{\\Gamma(k)}x^{k-1}e^{-x},
+
+    Args:
+        shape (array): Parameter of the gamma distribution :math:`k`.
+        size (int or tuple of ints): The shape of the array. If ``None``, a
+            zero-dimensional array is generated.
+        dtype: Data type specifier. Only :class:`numpy.float32` and
+            :class:`numpy.float64` types are allowed.
+
+    Returns:
+        cupy.ndarray: Samples drawn from the standard gamma distribution.
+
+    .. seealso::
+        :func:`cupy.random.RandomState.standard_gamma`
```

Comment:
I don't need the link from `cupy.random.standard_gamma` to `cupy.random.RandomState.standard_gamma`.  Do you want to have this?

Reply:
No, I don't. I think you are right. I'll remove.